src/views/Home.jsx

```diff
@@ -1,38 +1,34 @@
 import './Home.css';
 import { SingleList } from '../components';
-import { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { createList } from '../api';
+import TextInputElement from '../components/TextInputElement';
 
 export function Home({ data, setListPath, userId, userEmail }) {
-	const [listName, setListName] = useState('');
 	const navigate = useNavigate();
 
-	const handleChange = (event) => {
-		setListName(event.target.value);
-	};
-
```

Comment:
remove state object in favor of extracting form value directly from event object